redditdownload/redditdownload.py

```diff
@@ -382,177 +454,241 @@ def main():
     # compile reddit comment url to check if url is one of them
     reddit_comment_regex = re.compile(r'.*reddit\.com\/r\/(.*?)\/comments')
 
-    LAST = ARGS.last
-
     start_time = None
     ITEM = None
 
     sort_type = ARGS.sort_type
     if sort_type:
         sort_type = sort_type.lower()
 
-    while not FINISHED:
-        ITEMS = getitems(
-            ARGS.reddit, multireddit=ARGS.multireddit, previd=LAST,
-            reddit_sort=sort_type)
-
-        # measure time and set the program to wait 4 second between request
-        # as per reddit api guidelines
-        end_time = time.clock()
-
-        if start_time is not None:
-            elapsed_time = end_time - start_time
-
-            if elapsed_time <= 4:  # throttling
-                time.sleep(4 - elapsed_time)
-
-        start_time = time.clock()
+    if ARGS.subreddit_list:
+        ARGS.subreddit_list = ARGS.subreddit_list[0]
+        SUBREDDIT_FILE = os.path.join(os.getcwd(), ARGS.subreddit_list)
```

Comment:
variable not written in capital letters. old arguments are lot like this and yet not fixed. we don't want to continue that.